nest-forms-backend/src/files/files.service.ts

```diff
@@ -545,6 +545,19 @@ export default class FilesService {
         )
       }
 
+      /* delete file form scanner if contains scannerId */
+      if (file.scannerId) {
+        const deleteScannerStatus =
+          // eslint-disable-next-line no-await-in-loop
+          await this.filesHelper.deleteFileFromScannerClient(file.scannerId)
+        if (deleteScannerStatus === undefined) {
+          throw this.throwerErrorGuard.InternalServerErrorException(
```

Comment:
Are we sure there aren't legitimate cases where scanner don't know about the error and it will fail this request?

Reply:
this error in the normal cases should not happen as every file which are you going to delete already should have scannerId and therefore it was in registered in DB.